pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -56,9 +56,11 @@ type Reconciler struct {
 }
 
 var (
-	apiConfigMapName string = "api"
-	errKeyMissing    error  = fmt.Errorf("secret doesn't contains all the keys")
-	namespace        string
+	apiConfigMapName       string = "api"
```

Comment:
pls use `tekton-hub-` prefix for hub resources

Reply:
yeah i'll submit a patch for this once we have a release for hub where we are adding the prefix to all components